extension/test/integration/get-carbon-offset-costs.handler.spec.js

```diff
@@ -0,0 +1,125 @@
+const { expect } = require('chai')
+
+const ctpClientBuilder = require('../../src/ctp')
+const c = require('../../src/config/constants')
+const config = require('../../src/config/config')
+
+describe('get-carbon-offset-costs', () => {
+  let ctpClient
+  const adyenMerchantAccount = Object.keys(
+    JSON.parse(process.env.ADYEN_INTEGRATION_CONFIG).adyen
+  )[0]
+  const commercetoolsProjectKey = Object.keys(
+    JSON.parse(process.env.ADYEN_INTEGRATION_CONFIG).commercetools
+  )[0]
+
+  before(async () => {
+    const ctpConfig = config.getCtpConfig(commercetoolsProjectKey)
+    ctpClient = ctpClientBuilder.get(ctpConfig)
+  })
+
+  it('should calculate delivery offset', async () => {
+    const getCarbonOffsetCostsRequestDraft = {
+      originCountry: 'BE',
+      deliveryCountry: 'FR',
+      packageWeight: {
+        value: 2.2,
+        unit: 'kg',
+      },
+    }
+    const interfaceInteractionResponse = await getCarbonOffsetCosts(
+      getCarbonOffsetCostsRequestDraft
+    )
+    expect(interfaceInteractionResponse.deliveryOffset).to.exist
+    expect(interfaceInteractionResponse.totalOffset).to.exist
```

Comment:
why not do assertion for `deliveryOffset.value` as well? (Same in line 66-69)

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/857/commits/1870344d491abbb7f777e4ff7399bfbdfef41d4a